glean_parser/util.py

```diff
@@ -219,10 +217,9 @@ def resolve_remote(self, uri):
     return NullResolver.from_schema(schema)
 
 
-def fetch_remote_url(url, cache=True):
```

Comment:
I think we still want to cache.  This is important because caching means we don't have to be connected to the internet every time we run Glean unit tests.  Was there something that broke by not doing this?